etl/steps/data/garden/moatsos/2023-10-09/shared.py

```diff
@@ -0,0 +1,199 @@
+
+                tb_garden[col_name].metadata.description_short = tb_garden[col_name].metadata.description_short.replace(
+                    "{povline}", cbn_dict[cbn]["description"]
+                )
+
+            # For variables above CBN
+            col_name = f"{var}_above_{cbn}"
+
+            if col_name in cols:
+                # Create metadata for these variables
+                tb_garden[col_name].metadata = var_metadata_cbn(var, cbn)
+
+                tb_garden[col_name].metadata.description_short = tb_garden[col_name].metadata.description_short.replace(
+                    "{povline}", cbn_dict[cbn]["description"]
+                )
+
+                tb_garden[col_name].metadata.description_short = tb_garden[col_name].metadata.description_short.replace(
+                    "below", "above"
+                )
+                tb_garden[col_name].metadata.title = tb_garden[col_name].metadata.title.replace(
+                    "in poverty", "not in poverty"
+                )
+                tb_garden[col_name].metadata.display["name"] = tb_garden[col_name].metadata.title
+
+        for i in range(len(povline_list)):
+            if i != 0:
+                # For variables between poverty lines
+                col_name = f"{var}_between_{povline_list[i-1]}_{povline_list[i]}"
+
+                if col_name in cols:
+                    # Create metadata for these variables
+                    tb_garden[col_name].metadata = var_metadata_between(var, povline_list[i - 1], povline_list[i])
+
+                    tb_garden[col_name].metadata.description_short = tb_garden[
+                        col_name
+                    ].metadata.description_short.replace(
+                        "{povline}",
+                        f"living between {povline_dict[povline_list[i-1]]['title_between']} and {povline_dict[povline_list[i]]['title_between']} a day",
+                    )
+
+    return tb_garden
+
+
+# Metadata functions to show a clearer main code
+def var_metadata_absolute(var, povline) -> VariableMeta:
+    meta = VariableMeta(
+        title=f"{povline_dict[povline]['title']} - {var_dict[var]['title']}",
+        description_short=f"""{var_dict[var]['description']}""",
+        description_key=f"""
+        - {ppp_description}
+        - {dod_description}""",
+        unit=var_dict[var]["unit"],
+        short_unit=var_dict[var]["short_unit"],
+    )
+    meta.display = {
+        "name": meta.title,
+        "numDecimalPlaces": var_dict[var]["numDecimalPlaces"],
+        "entityAnnotationsMap": "Western offshoots (Moatsos): United States, Canada, Australia and New Zealand",
+    }
+
+    return meta
+
+
+def var_metadata_between(var, povline1, povline2) -> VariableMeta:
+    meta = VariableMeta(
+        title=f"{povline_dict[povline1]['title_between']}-{povline_dict[povline2]['title_between']} - {var_dict[var]['title']}",
+        description_short=f"""{var_dict[var]['description']}""",
+        description_key=f"""
```

Comment:
@pabloarosado Another quick question: When adding metadata by code, how can I add `description_key` fields to see them as a list?